extensions/replication/tasks/UpdateReplicationStatus.js

```diff
@@ -102,6 +115,9 @@ class UpdateReplicationStatus extends BackbeatTask {
             score,
         };
         this.failedCRRProducer.publishFailedCRREntry(JSON.stringify(message));
+        if (this.bucketNotificationConfig) {
```

Comment:
this function may not be called, e.g. if `repConfig.monitorReplicationFailures` is not set : but we should still post a bucket notification in that case.